src/ostree/ot-builtin-refs.c

```diff
@@ -60,7 +70,60 @@ static gboolean do_ref (OstreeRepo *repo, const char *refspec_prefix, GCancellab
           g_print ("%s\n", ref);
         }
     }
+  else if(opt_create)
+    {
+      gboolean old_ref_exists = FALSE;
+      const char *checksum = NULL;
+
+      g_hash_table_iter_init (&hashiter, refs);
+      while (g_hash_table_iter_next (&hashiter, &hashkey, &hashvalue))
```

Comment:
The reason I did that is because delete does it that way. Should I change that for delete as well?